package-real.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "author": "Simon Lydell",
   "repository": "lydell/eslint-plugin-simple-import-sort",
-  "description": "Easy autofixable import sorting",
+  "description": "Easy, autofixable import sorting",
```

Comment:
I asked a _very_ trustworthy source called ChatGPT 🙃  for advice on this:

> Both can be correct because they convey slightly different nuances:
>
> 1. **“Easy, autofixable import sorting”**: The comma separates "easy" and "autofixable," suggesting that the import sorting is both easy and autofixable as two distinct qualities.
>
> 2. **“Easy autofixable import sorting”**: Without the comma, "easy" and "autofixable" work together as a single phrase to describe the import sorting, implying that the process of making imports autofixable is easy.
>
> While the comma adds a slight pause, both forms are grammatically correct and convey a similar overall meaning. The choice depends on whether you want to emphasize the qualities separately or together.

I’m not a native English speaker, so I can’t tell what’s right.

Either way, if we should go with the comma, we should make the same change in the readme.

Reply:
I think ChatGPT is right about that.
I would've said it was both easy (as in it makes sense) and autofixable, but it could also be easily autofixable, in which case its fine as is.